core/consensus/slots/src/slots.rs

```diff
@@ -38,6 +38,11 @@ pub fn duration_now() -> Option<Duration> {
 	}).ok()
 }
 
+/// Get the slot for now.
+pub fn slot_now(slot_duration: u64) -> Option<u64> {
+	duration_now().map(|s| s.as_secs() / slot_duration)
```

Comment:
The struct `SlotDuration` specifies time in milliseconds, is there a unit error here? If you are expecting seconds as the `slot_duration` input, I think the doc comment should specify that it expects an input in seconds.